### PR TITLE
Add Made with Prisma badge

### PR DESCRIPTION
**Pull request by petradonka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +172,20 @@ The projects in the [`deployment-platforms`](./deployment-platforms) directory s
 - [Indie Stack](https://github.com/remix-run/indie-stack): Remix Stack for deploying to Fly with SQLite, authentication, testing, linting, formatting
 - [Blues Stack](https://github.com/remix-run/blues-stack): Remix Stack for deploying to Fly with PostgreSQL, authentication, testing, linting, formatting
 
+## Badges
+
+[![Made with Prisma](http://made-with.prisma.io/dark.svg)](https://prisma.io) [![Made with Prisma](http://made-with.prisma.io/indigo.svg)](https://prisma.io)
+
+Built something awesome with Prisma? 🌟 Show it off with these [badges](https://github.com/prisma/presskit?tab=readme-ov-file#badges), perfect for your readme or website.
```
Comment (nikolasburk) [inline README.md:179]: I see README typically spelled being all-caps:

```suggestion
Built something awesome with Prisma? 🌟 Show it off with these [badges](https://github.com/prisma/presskit?tab=readme-ov-file#badges), perfect for your README or website.
```

File: README.md [hunk 2]
```diff
@@ -172,6 +172,20 @@ The projects in the [`deployment-platforms`](./deployment-platforms) directory s
...
+
+```
```
Comment (nikolasburk) [inline README.md:181]: we could use a markdown annotation for syntax highlighting here:

````suggestion
```md
````

File: README.md [hunk 3]
```diff
@@ -172,6 +172,20 @@ The projects in the [`deployment-platforms`](./deployment-platforms) directory s
...
+[![Made with Prisma](http://made-with.prisma.io/dark.svg)](https://prisma.io)
+```
+
+```
```
Comment (nikolasburk) [inline README.md:185]: we could use a markdown annotation for syntax highlighting here:

````suggestion
```md
````

**Review by nikolasburk — APPROVED** [review]
I've left some small suggestions for consideration, none of these blockers to merge 👍

File: README.md (hunk 2 above)
Comment (petradonka) [reply README.md:181]: Thanks for the suggestion! I've tried this on presskit and just didn't see any difference 😄 Also expected syntax highlighting. We can add this later if needed! 👌

**Pull request merged**